docs/changelog.rst

```diff
@@ -44,6 +45,37 @@ Core API
   - :attr:`mopidy.core.TracklistController.repeat`
   - :attr:`mopidy.core.TracklistController.single`
 
+- Removed the deprecated method
+  :meth:`mopidy.core.LibraryController.find_exact`.
+  Use :meth:`~mopidy.core.LibraryController.search`
+  with the keyword argument ``exact=True`` instead.
+  (Fixes: :issue:`1461`, PR: :issue:`1769`)
+
+- Removed deprecated methods from the playback controller.
```

Comment:
Is this better summarized as removing getters that have been moved to mixer controller?

Reply:
Yeah, I think so. Currently the changelog is a bit too much organized around what arrived in which PR. I think the deprecated/removed stuff should be organized by which controller the change is in. I'll reorganize that.